tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,327 @@
+        },
+        {
+            "address": "carlosaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+                {
+            "address": "charlieaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+    ]
+}
+```
+
+```sh
+simd tx group update-group-members $ALICE_KEY 1 members.json
+```
+
+We can verify that the group members are updated:
+
+```sh
+simd query group group-members 1
+```
+
+As an exercice, please re-add bob in the group and go to the next section.
+
+## Create group policy
+
+Next we need to decide of a group policy. This defines how long a proposal can be voted on and how the outcome is calculated.
+Here we use the `ThresholdDecisionPolicy`, it defines a threshold of yes votes (based on a tally of voter weights) that must be achieved in order for a proposal to pass.
+
+Following is the content of the `policy.json`. Proposals can be voted on for a maximum of 10 minutes.
+A proposal requires only one vote to pass.
+
+```json
+{
+    "@type": "/cosmos.group.v1.ThresholdDecisionPolicy",
+    "threshold": "1",
+    "windows": {
+        "voting_period": "10m",
+        "min_execution_period": "0s"
+    }
+}
+```
+
+
+```sh
+simd tx group create-group-policy $ALICE_KEY 1 "" policy.json
+```
+
+Let's verify our newly created group policy and save its address for future use:
+
+```sh
+simd query group group-policies-by-group 1
+export GROUP_POLICY_ADDRESS=$(simd query group group-policies-by-group 1 --output json | jq -r '.group_policies[0].address')
+```
+
+## Vote on proposal
+
+Now that we have a group with a few members and a group policy, let's submit our first group proposal.
+Like for members management, we need to create a `proposal.json` file that contains the proposal.
+
+A proposal can be of any message. For this tutorial, we continue with our example of an association.
+The treasurer, Bob, wants to send money to a third party to pay the bills and creates a `proposal.json`:
+
+
+```json
+{
+    "group_policy_address": "cosmos1..." , // the $GROUP_POLICY_ADDRESS
+    // array of proto-JSON-encoded sdk.Msgs
+    "messages": [
+        {
+            "@type": "/cosmos.bank.v1beta1.MsgSend",
+            "from_address": "cosmos1...", // the $GROUP_POLICY_ADDRESS
+            "to_address": "cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad", // a third-party
+            "amount": [
+                {
+                    "denom": "stake",
+                    "amount": "100"
+                }
+            ]
+        }
+    ],
+    "metadata": "costs utilities",
+    "proposers": [ "cosmos1..." ] // $BOB_KEY
+}
+```
+
+This proposal, if pass, will send 100 stake to cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad to pay the bills.
+The tokens will be sent from the decision policy address.
+
+::: tip
+The decision policy has no fund yet. We can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.
+:::
```

Comment:
```suggestion
<HighlightBox type"tip">

The decision policy has no fund yet. We can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.

</HighlightBox>
```
Can you explain what happens if there are 2 messages in the proposal and one fails at execution?